README.md

```diff
@@ -1,198 +1,233 @@
-
-see below for more hands on examples.
+## Supported Engines
+
+| Engine                                                                       | Class name              |
+|------------------------------------------------------------------------------|-------------------------|
+| [Google Search Engine](https://serpapi.com/search-api)                       | `GoogleSearch()`        |
+| [Google Maps](https://serpapi.com/google-maps-api)                           | `GoogleSearch()`        |
+| [Google Jobs](https://serpapi.com/google-jobs-api)                           | `GoogleSearch()`        |
+| [Google Trends](https://serpapi.com/google-trends-api)                       | `GoogleSearch()`        |
+| [Google Autocomplete](https://serpapi.com/google-autocomplete-api)           | `GoogleScholarSearch()` |
+| [Google Related Questions](https://serpapi.com/google-related-questions-api) | `GoogleScholarSearch()` |
+| [Google Scholar](https://serpapi.com/google-scholar-api)                     | `GoogleScholarSearch()` |
+| [Google Play Store](https://serpapi.com/google-play-api)                     | `GoogleSearch()`        |
+| [Google Product](https://serpapi.com/google-product-api)                     | `GoogleSearch()`        |
+| [Google Immersive Product](https://serpapi.com/google-immersive-product-api) | `GoogleSearch()`        |
+| [Google Reverse Image](https://serpapi.com/google-reverse-image)             | `GoogleSearch()`        |
+| [Google Events](https://serpapi.com/google-events-api)                       | `GoogleSearch()`        |
+| [Google Local Services](https://serpapi.com/google-local-services-api)       | `GoogleSearch()`        |
+| [Bing](https://serpapi.com/bing-search-api)                                  | `BingSearch()`          |
+| [Baidu](https://serpapi.com/baidu-search-api)                                | `BaiduSearch()`         |
+| [DuckDuckGo](https://serpapi.com/duckduckgo-search-api)                      | `DuckDuckGoSearch()`    |
+| [Yahoo](https://serpapi.com/yahoo-search-api)                                | `YahooSearch()`         |
+| [Yandex](https://serpapi.com/yandex-search-api)                              | `YandexSearch()`        |
+| [eBay](https://serpapi.com/ebay-search-api)                                  | `EbaySearch()`          |
+| [Youtube](https://serpapi.com/youtube-search-api)                            | `YoutubeSearch()`       |
+| [Walmart](https://serpapi.com/walmart-search-api)                            | `WalmartSearch()`       |
+| [HomeDepot](https://serpapi.com/home-depot-search-api)                       | `HomeDepotSearch()`     |
+| [Apple App Store](https://serpapi.com/apple-app-store)                       | `AppleAppStoreSearch()` |
+| [Naver](https://serpapi.com/naver-search-api)                                | `NaverSearch()`         |
+| [Yelp](https://serpapi.com/yelp-search-api)                                  | `YelpSearch()`          |
```

Comment:
`Supported Engines` table is 990 px tall. (Not fitting one screen with 1080p resolution.)

![image](https://user-images.githubusercontent.com/282605/213187278-364caa02-d750-4f00-832f-5b8f4ebb3f62.png)

Reply:
@ilyazub should we keep it? Possibly it would better look and be easier to read if we wrap it with the `<details>` element like so:

<details>
<summary>Google Search Engine and 10+ engines...</summary>

| Engine                                                                       | Class name              |
|------------------------------------------------------------------------------|-------------------------|
| [Google Search Engine](https://serpapi.com/search-api)                       | `GoogleSearch()`        |
| [Google Maps](https://serpapi.com/google-maps-api)                           | `GoogleSearch()`        |
| [Google Jobs](https://serpapi.com/google-jobs-api)                           | `GoogleSearch()`        |
| [Google Trends](https://serpapi.com/google-trends-api)                       | `GoogleSearch()`        |
| [Google Autocomplete](https://serpapi.com/google-autocomplete-api)           | `GoogleScholarSearch()` |
| [Google About This Result](https://serpapi.com/google-about-this-result)     | `GoogleSearch()`        |
| [Google Lens](https://serpapi.com/google-lens-api)                           | `GoogleSearch()`        |
| [Google Finance](https://serpapi.com/google-finance-api)                     | `GoogleSearch()`        |
| [Google Related Questions](https://serpapi.com/google-related-questions-api) | `GoogleScholarSearch()` |
| [Google Scholar](https://serpapi.com/google-scholar-api)                     | `GoogleScholarSearch()` |
| [Google Play Store](https://serpapi.com/google-play-api)                     | `GoogleSearch()`        |
| [Google Product](https://serpapi.com/google-product-api)                     | `GoogleSearch()`        |
| [Google Immersive Product](https://serpapi.com/google-immersive-product-api) | `GoogleSearch()`        |
| [Google Reverse Image](https://serpapi.com/google-reverse-image)             | `GoogleSearch()`        |
| [Google Events](https://serpapi.com/google-events-api)                       | `GoogleSearch()`        |
| [Google Local Services](https://serpapi.com/google-local-services-api)       | `GoogleSearch()`        |
| [Bing](https://serpapi.com/bing-search-api)                                  | `BingSearch()`          |
| [Baidu](https://serpapi.com/baidu-search-api)                                | `BaiduSearch()`         |
| [DuckDuckGo](https://serpapi.com/duckduckgo-search-api)                      | `DuckDuckGoSearch()`    |
| [Yahoo](https://serpapi.com/yahoo-search-api)                                | `YahooSearch()`         |
| [Yandex](https://serpapi.com/yandex-search-api)                              | `YandexSearch()`        |
| [eBay](https://serpapi.com/ebay-search-api)                                  | `EbaySearch()`          |
| [Youtube](https://serpapi.com/youtube-search-api)                            | `YoutubeSearch()`       |
| [Walmart](https://serpapi.com/walmart-search-api)                            | `WalmartSearch()`       |
| [HomeDepot](https://serpapi.com/home-depot-search-api)                       | `HomeDepotSearch()`     |
| [Apple App Store](https://serpapi.com/apple-app-store)                       | `AppleAppStoreSearch()` |
| [Naver](https://serpapi.com/naver-search-api)                                | `NaverSearch()`         |
| [Yelp](https://serpapi.com/yelp-search-api)                                  | `YelpSearch()`          |

</details>

WDYT?